Apps/WebClient/src/ClientApp/src/components/report/covid19.vue

```diff
@@ -73,6 +83,20 @@ export default class COVID19ReportComponent extends Vue {
         return this.visibleRecords.length == 0;
     }
 
+    private get items(): LaboratoryRow[] {
+        return this.visibleRecords.map<LaboratoryRow>((x) => {
+            const labResult = x.labResults[0];
+            return {
+                date: DateWrapper.format(labResult.collectedDateTime),
```

Comment:
Pretty interesting, didn't know that character existed. Since DateWrapper.format is used pretty much everywhere I created a story to do that and make sure it does not break anything else
https://qslvic.visualstudio.com/Health%20Gateway/_sprints/taskboard/Health%20Gateway%20Team/Health%20Gateway/Release%201.4/Sprint%202?workitem=10601